db/seeds.rb

```diff
@@ -5,3 +5,15 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+if Doorkeeper::Application.count.zero?
+  Doorkeeper::Application.create!(name: "Web Client", redirect_uri: "", scopes: "")
+  Doorkeeper::Application.create!(name: "iOS Client", redirect_uri: "", scopes: "")
+  Doorkeeper::Application.create!(name: "Android Client", redirect_uri: "", scopes: "")
+end
+
+User.first_or_create(
+  email: "test@nimblehq.co",
```

Comment:
```suggestion
User.find_or_create_by(email: 'test@nimblehq.co')
```

In case your DB already contains a user with another email, you might not understand why your API can't authenticate even right after generating all the seeds successfully.

With this syntax, if will check if the user with the `test@nimblehq.co` email already exists or not. If not, it will create it ;-)